Library/Formula/cassandra.rb

```diff
@@ -28,23 +64,26 @@ def install
 
     rm Dir["bin/*.bat", "bin/*.ps1"]
 
+    rm_rf HOMEBREW_PREFIX/"etc/cassandra" # Error: File exists - /usr/local/etc/cassandra/triggers
```

Comment:
I'm fairly sure this *has* to be our error somewhere - I thought the ` etc.install ` method was supposed to handle overwrites in a friendly way, so we shouldn't be seeing the error. I pretty much put the ` rm_rf ` in place until you or another maintainer went "Huh, this can be fixed by this xyz" :smile_cat: - It stumped me.